README_IT.md

```diff
@@ -99,8 +99,6 @@ Invitiamo tutti a partecipare al gruppo su Slack per comunicare al meglio su que
 
 # FAQ (Sezione in fase di sviluppo, restate sintonizzati! Abbiamo bisogno di volontari che contribuiscano a questa sezione, chi è interessato è invitato a contattarci!)
 
-FAQ generiche sul progetto finora raccolte: [clicca qui](./FAQ.md).
-
```

Comment:
Now the new url is here :https://community.wuhan2020.org.cn/it-it/docs/overview/faq.html